dashboard/templates/dashboard/applications_list.html

```diff
@@ -18,13 +18,13 @@
                 </div>
             </div>
 
-    {% if application_type == "ap" %}
+    {% if application_type == Application.STATUS_APPLIED %}
```

Comment:
@surajgovardhangaikwad I was referring to this about the scattered string constants.